scripts/ci/install_deps.sh

```diff
@@ -15,7 +15,10 @@ wget https://get.helm.sh/helm-"${HELM_VERSION}"-linux-amd64.tar.gz -O /tmp/helm.
 tar xzf /tmp/helm.tar.gz -C /tmp --strip-components=1 && chmod +x /tmp/helm && sudo mv /tmp/helm /usr/local/bin
 
 if [[ "$REQUIRES_MINIKUBE" == "true" ]]; then
-    # Download minikube.
+  sudo apt-get update
+  sudo apt-get install -y conntrack
```

Comment:
Is this now needed by Minikube, or does it enable some diagnostics?

Reply:
by k8s